src/Application.php

```diff
@@ -187,8 +178,9 @@ public function __call($method, $args)
                 ));
         }
 
-        // @TODO: we can use variadic parameters when dependency is raised to PHP 5.6
-        return call_user_func_array([$this, 'route'], $args);
+        $callback = [$this, 'route'];
+
+        return $callback(...$args);
```

Comment:
Can we have here just `$this->route(... $args)`? Isn't it the same?

BTW. About coding standards - should be space after `...` or not?